src/View/mainpage/DrawROIWindow/Drawing.py

```diff
@@ -392,6 +394,16 @@ def set_source(self, event):
         self.fill_source = [round(event.scenePos().x()), round(event.scenePos().y())]
         self._display_pixel_color()
 
+    @Slot(bool)
+    def set_is_drawing(self, is_drawing):
+        """
+        Function triggered when either the Draw or Fill button is pressed from the menu.
+        Sets the is_drawing bool, allows for filling or drawing from separate button presses
+        """
+        logging.debug("set_is_drawing started")
+        self.is_drawing = is_drawing
+        logging.debug("set_is_drawing finished")
```

Comment:
probably don't need a finished

Reply:
same with most, however check with ryan. My worry is that we will end up with a ton of logs.

Reply:
If it is actually relevent to debugging the process then leave it. But in this case the extra log on line 405 would not help in any type of debugging as the line 404 should never fail. The first log might be useful just to know when that function is being called when debugging.

Reply:
Cheers for feedback, will resolve by removing the logging on line 405